Sources/GRPC/ConnectionKeepalive.swift

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import NIO
+
+/// Provides keepalive pings.
+///
+/// The defaults are determined by the gRPC keepalive
+/// [documentation] (https://github.com/grpc/grpc/blob/master/doc/keepalive.md).
+public struct ClientConnectionKeepalive {
```

Comment:
These two structures should be conformed to `Hashable`, as they're just simple aggregates.